common/language.c

```diff
@@ -277,6 +275,22 @@ const char* const g_strings[StrId_Max][16] =
         STR_ZH("打开"),
         STR_TW("打开"),
     },
+    
+    [StrId_Actions_Back] =
+    {
+        STR_EN("Back"),
+        STR_ES("Espalda"),
```

Comment:
lol nope, this means "Back" as in body part! This should be "Atrás". Please do not use google translate; get actual translators to do the job or just leave STR_EN and the others untranslated...